src/main/java/sparksoniq/semantics/DynamicContext.java

```diff
@@ -202,6 +202,9 @@ public Item getLocalVariableCount(String varName) {
         if (_localVariableCounts.containsKey(varName)) {
```

Comment:
The name of this function could simply by getVariableCount(). It just returns the count irregardless of whether the variable is local, RDD or DF.

Reply:
Very nice ! I like the name and the functionality